Разметка внутренней страницы

aidamgm opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1134435/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/1134435-sedona-33/1/

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1134435/tasks/4)

aidamgm committed:
Update index.html

разметка главной страницы

aidamgm committed:
Update Readme.md

еще раз выбрала наставника

aidamgm committed:
Create Catalog.html

каталог

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+        </ul>
+          <li class="navigation-item">
+            <button class="navigation-show" type="button">Хочу сюда</button>
+          </li>
```
li может быть только внутри списка, тут лучше вообще убрать тег li

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+              <br> своя жизнь</p>
+              <img class="first-reasons" src="" width="800" height="384" alt="фото гостиницы">
+              <ul class="advices-list">
+                <h4 class="visually-hidden">Советы</h4>
```
Прямыми потомками ul могут быть только li. Стоит убрать заголовок.
Проверить страницу в валидаторе.

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+              <ul class="advices-list">
+                <h4 class="visually-hidden">Советы</h4>
+                <li class="services-item">
+                  <h3 class="advices-title-home">Жильё</h3>
```
Если у родительского блока был заголовок третьего уровня, то тут должен быть четвертого.

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+        </section>
+        
+        <section class="appointment">
+          <h2 class="appointment-title">Заинтересовались?</h2>
```
Эта фраза не несет смысла, не стоит ее делать заголовком

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+          <br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          <form class="appointment-form" action="https://echo.htmlacademy.ru/" method="post">
+            <button class="button" type="submit">ПОИСК ГОСТИНИЦЫ В седоне</button>
+          </form>
```
Это просто кнопка, без формы и тип кнопки стоит поменять

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+          <label for="newsletter-email"></label>
+          <input class="field"  placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email" required>
+          <button class="button newsletter-button" type="submit">
+            <span class="visually-hidden">Подписаться</span>
```
visually-hidden - класс, который скроет подпись, а по макету подпись должна быть видна.

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+          <p class="subscription">Только полезная информация и никакого спама,
+          <br>честное бойскаутское!</p>
+          <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="newsletter-email"></label>
```
Не стоит оставлять лейбл пустым. От такого лейбла нет толку.

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+              <br>туристов</h3>
+              <span class="fifth">— №5—</span>
+              <p class="fifth-reasons-description">Большинство едет в Гранд Каньон и толпится там</p>
+            </li>
```
Стоит обсудить классы.
В этом списке я вижу классы first, second, third, fourth, fifth.
Не стоит раздавать разные классы элементам, у которых будет одинаковая стилизация.
К примеру, fourth-reasons-description и fifth-reasons-description по стилям не имеет различий, если писать стилизацию по разным классам, то стилей будет больше.

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+        
+        <section class="appointment">
+          <h2 class="appointment-title">Заинтересовались?</h2>
+          <p class="appointment">Укажите предполагаемые даты поездки,
```
И периодически встречаются одинаковые классы на разных элемента. К примеру, тут на секшене стоит класс appointment и такой же класс стоит на абзаце. 
Абзацу можно поменять класс на appointment-text или appointment-description

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="ru">
```
В названиях файлов нельзя использовать заглавные буквы, поэтому Catalog.html нужно заменить на catalog.html

Komnatnaya replied:
Название файла не поменялось, попробуй удалить этот файл, закоммитить результат, потом вернуть и опять закоммитить. Либо поменяй название полностью, к примеру на hotels.html

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+    </header>
+    <main class="main-inner">
+        <header class="inner-header">
+            <h1 class="inner-header-title">Каталог</h1>
```
По макету текст в заголовке другой.

Komnatnaya replied:
"По макету текст в заголовке другой."
Не исправлено

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                        Мотель
+                    </label>
+                    <label class="control">
+                        <input class="control-input" type="radio" name="product-group" value="type-house">
```
У радио-кнопок должны быть одинаковые name и разные value, иначе они отправлять на сервер одно и то же в независимости от выбора пользователя.

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                  </ul> 
+            </fieldset>
+            <button class="button" type="submit">Применить</button>
+                <a class="no-filter" href="#">Сбросить</a>
```
Сбросить - это reset http://htmlbook.ru/html/button/type

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+            <button class="button" type="submit">Применить</button>
+                <a class="no-filter" href="#">Сбросить</a>
+              </form>
+              <select class="select-control" aria-label="Сортировка">
```
Нет разметке этого элемента
![image](https://user-images.githubusercontent.com/20788716/152521601-5adaf546-75e6-477d-ae70-588cb282cd85.png)

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                      <h3 class="product-card-title">Amara Resort & Spa</h3>
+                    </a>
+                    <span class="catalog-type-hotels">Гостиница</span>
+                    <b class="product-card-price">От 4000 ₽</b>
```
Неверно использован тег, в теге b текст становится жирным, а по макету не должен быть таким.

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                    <b class="product-card-price">От 4000 ₽</b>
+                    <a class="product-card-about button" href="#">Подробнее</a>
+                    <button class="button" type="submit">В избранное</button>
+                    <span class="visually-hidden">&#9733;&#9733;&#9733;&#9733;</span>
```
Звездочки будет добавлять стилями на стадии декора, а сейчас в спан с классом visually-hidden можно положить подпись, сколько звезд у отеля.

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                    <li class="pagination-item">
+                      <a class="pagination-link pagination-prev pagination-disabled">
+                        <span class="visually-hidden">Предыдущая страница</span>
+                      </a>
```
В макете нет такого элемента

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                      </a>
+                    </li>
+                  </ul>
+                  <button class="button pagination-show-more" type="button">Загрузить ещё</button>
```
Кнопка "Загрузить ещё" по макету должна стоят до пагинации

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,261 @@
+                  </ul>
+                  <button class="button pagination-show-more" type="button">Загрузить ещё</button>
+          </section>
+        </main>
```
Нет разметки этого элемента
![image](https://user-images.githubusercontent.com/20788716/152523007-b14c5316-4f99-43f6-9a1d-0e661dd41da7.png)

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,158 @@
+            <a class="navigation-link" href="about.html">О Седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="hotels.html">Гостиницы</a>
```
Чтобы можно было переходить по страницам в href нужно прописать название второй страницы

aidamgm committed:
Update index.html

все исправила, кроме href - я не поняла что нужно прописать для второй страницы

aidamgm committed:
Update Catalog.html

в  vsc исправила имя на "catalog.html" но в гит хабе стоит также с заглавной буквы

Komnatnaya commented on index.html:
```diff
@@ -2,11 +2,152 @@
+                  <br>всё как в кино!</span>
+                </li>
+                <li class="services-item">
+                  <h3 class="advices-title-food">Еда</h3>
```
"Если у родительского блока был заголовок третьего уровня, то тут должен быть четвертого." 
Эта правка касается всех одинаковых блоков, не только Жилье, но и Еда и Сувениры.

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,262 @@
+                    <span class="catalog-type-hotels">Гостиница</span>
+                    <p class="product-card-price">От 4000 ₽</p>
+                    <a class="product-card-about button" href="#">Подробнее</a>
+                    <button class="button" type="submit">В избранное</button>
```
submit - это кнопка отправки формы. Если нет формы, то стоит использовать type=button

Komnatnaya commented on Catalog.html:
```diff
@@ -0,0 +1,262 @@
+                    </li>
+                  </ul>
+                  <span class="view-item">Гостиниц на странице:</span>
+                    <button class="select-view">4</button>
```
Это выпадающий список select, такой же как Сортировка, только тут пользователь выбирать количество выводимых товаров.

aidamgm committed:
Update index.html

aidamgm committed:
Create hotels.html

 каталогу пришлось поменять название
